File: abc086-a/java/Main.java
import java.util.Scanner;

public class Main {

    private static boolean isEven(int n) {
        return n % 2 == 0;
    }

    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {
            int a = scanner.nextInt();
            int b = scanner.nextInt();
            String output = isEven(a) || isEven(b) ? "Even" : "Odd";
            System.out.println(output);
        }
    }
}

File: abc086-a/js/main.js
function Main(input) {
    const [a, b] = input.split("\n")[0].split(" ").map(Number);
    const output = (a % 2 === 0 || b % 2 === 0) ? "Even" : "Odd";
    console.log(output);
}

Main(require("fs").readFileSync("/dev/stdin", "utf8"));
